scout/server/links.py

```diff
@@ -420,7 +420,7 @@ def ensembl_link(variant_obj, build=37):
     """Compose (sv) variant link to ensembl"""
 
     my_end = variant_obj["end"]
-    if variant_obj["chromosome"] != variant_obj.get("end_chrom"):
+    if variant_obj["chromosome"] != variant_obj.get("end_chrom", variant_obj["chromosome"]):
```

Comment:
Eh, sorry, I'm missing something? Are you trying to avoid some implicit None comparison issue or what?

Reply:
Ah ok. Fine. That is probably better - anything would do really, but yes.

Reply:
You could also check if variant_obj.get("end_chrom") was something first, which is sort of what we normally do?

Reply:
..but this works.

Reply:
I don't know, I just assumed that when end_chrom is missing it should be end_chrom = chromosome. This expression is more clear i think, but you are right, the PR would have worked anyway.

The expression should implicitly first check if variant_obj.get("end_chrom") and of it's not then end_chrom = chromosome, no?

Reply:
I think both would have worked really: if end_chrom == None, my_end=pos, if end_chrom != chr, my_end = pos but if end_chrom == chr my_end=end? In a way I guess the first one is then an easier read, but it is fine this way too.